src/NServiceBus.Core/ServicePlatform/Retries/MarkAsAcknowledgedBehavior.cs

```diff
@@ -17,9 +18,9 @@ public Task Invoke(IAuditContext context, Func<IAuditContext, Task> next)
         return next(context);
     }
 
-    public class State
+    public sealed class State
```

Comment:
Why make this state class sealed? Should we go ahead and do this for all of them?

Reply:
yeah change it back. I jumped the gun.